src/Provider/Vkontakte.php

```diff
@@ -21,53 +21,51 @@
  *       'callback'  => Hybridauth\HttpClient\Util::getCurrentUrl(),
  *       'keys'      => ['id' => '', 'secret' => ''],
  *   ];
- *
  *   $adapter = new Hybridauth\Provider\Vkontakte($config);
- *
  *   try {
  *       if (!$adapter->isConnected()) {
  *           $adapter->authenticate();
  *       }
- *
  *       $userProfile = $adapter->getUserProfile();
- *   }
- *   catch(\Exception $e) {
+ *   } catch (\Exception $e) {
  *       print $e->getMessage() ;
  *   }
  */
 class Vkontakte extends OAuth2
 {
+    const API_VERSION = '5.101';
+    
     /**
      * {@inheritdoc}
      */
     protected $apiBaseUrl = 'https://api.vk.com/method/';
 
     /**
      * {@inheritdoc}
+     * @see https://vk.com/dev/authcode_flow_user
      */
-    protected $authorizeUrl = 'https://api.vk.com/oauth/authorize';
+    protected $authorizeUrl = 'https://oauth.vk.com/authorize';
 
     /**
      * {@inheritdoc}
      */
-    protected $accessTokenUrl = 'https://api.vk.com/oauth/token';
+    protected $accessTokenUrl = 'https://oauth.vk.com/access_token';
 
     /**
      * {@inheritdoc}
+     * @todo Overwrite methods setUserStatus/getUserActivity(status), getUserPages/setPageStatus(pages), 
```

Comment:
We do not need this here in PR.

Reply:
@ApacheEx why it's normal practice

Reply:
normal practice where? for all "future" changes please create separate PR and describe goal of your changes.

Reply:
@ApacheEx [everywhere](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#515-todo)